apps/balancer-tools/src/app/apr/(utils)/sortChartData.ts

```diff
@@ -0,0 +1,34 @@
+interface iData {
+  x: number[] | string[];
+  y: number[] | string[];
+}
+
+/**
+ * Sorts data arrays by the values in the 'x' array.
+ *
+ * @param {Object} data - An object containing 'x' and 'y' arrays to be sorted.
+ * @returns {Array} An array containing sorted 'x' and corresponding 'y' arrays.
+ */
+export default function sortDataByX(data: iData) {
```

Comment:
Not really sure this function is needed. We just need to zip the array and order by one of two values right?
```suggestion
let x,y = [[1,2,3,...],[5,4,3....];
x.map((e, i) => [e, y[i]]).sort((x,y) => x[0]) # or y[0], whatever you want to do;
```